tools/pipelines/build-client.yml

```diff
@@ -97,3 +98,16 @@ extends:
           node node_modules/@fluidframework/build-tools/dist/genMonoRepoPackageJson/genMonoRepoPackageJson.js
           cp repo-package.json packages/package.json
           cp repo-package-lock.json packages/package-lock.json
+
+    additionalTests:
+      - task: Npm@1
+        displayName: '[end-to-end tests] npm run ci:test:realsvc'
+        env:
+          fluid__webpack__bearerSecret: $(fluid__webpack__bearerSecret)
+          fluid__webpack__fluidHost: $(fluid__webpack__fluidHost)
+          fluid__webpack__tenantId: $(fluid__webpack__tenantId)
+          fluid__webpack__tenantSecret: $(fluid__webpack__tenantSecret)
```

Comment:
These variables don't resolve properly, I assume because they're not resolved until in `build-npm-package.yml` and the values from the ADO UX aren't available anymore...?

Not sure the right approach. I don't really want to pollute `build-npm-package.yml`'s interface with additional parameters.

Reply:
Maybe I can add a parameter like this:
```
- name: secrets
  type: object
```

And then in the step I'm injecting do `${{ parameters.secrets.fluid__webpack__bearerSecret }}` or something.  Not sure how objects work exactly.